operations/research-and-development/engineering/cloud-engineering/README.md

```diff
@@ -1 +1,85 @@
-This page provides an overview of key processes and practices used by the Cloud Engineering Team at Mattermost.
+# Infrastructure
+
+## Mission
+
+The Infrastructure group empowers Mattermost to provide a SaaS Platform as Product which serves internal and external users by guaranteeing that we operate an enterprise-grade SaaS platform with self-serve powers.
+
+The Infrastructure group achieves this by focusing on **quality**, **availability**, **reliability**, **scalability**, and **security** objectives. In addition to this we prioritize cost efficiency and awareness adopting FinOps culture, which is strengthened by appropriately prioritized dogfooding initiatives.
+
+For the success of the SaaS Platform as a Product, there are many other teams which also contribute. The responsibility of the Infrastructure group is to evolve the SaaS platform enabled by platform observability data. 
+
+## Vision
+
+Operate fast, secure and reliable SaaS platform in which everyone can contribute
+
+## Teams
+
+- [SRE](/operations/research-and-development/organization/sre.md)
+- [Delivery](/operations/research-and-development/organization/cloud_platform.md)
+- [Platform](/operations/research-and-development/organization/delivery.md)
+
+## Principles
+
+- Be open and data driven
+- Use our own product to complete our mission
+- Align our strategy with the industry trends, company direction and customer needs
+- Influence and educate best practices
+
+## Design
+The Infrastructure group uses RFCs - requests for comment - or Design Docs as a common tool to describe the problem we are solving and represent the current state for any topic.
+
+## Dogfooding
```

Comment:
I missed to ask during our team review but I wonder if this could be:
```suggestion
## Internal Adoption
```

Reply:
I used Dogfooding as a wording because features has been created through our adoption. eg.
- Incident Management template in playbooks
- Playbooks two-ways actions for Gitlab checklists

Reply:
Sounds good to me, thanks!